publications-2015.bib

```diff
@@ -567,6 +567,17 @@ @Article{2015:grieshaber.mcbride.ea:uniformly
   pages   = {2255--2278}
 }
 
+@InProceedings{2015:hai.bause:adaptive,
+  doi     = {10.1115/imece2015-53265},
+  url     = {https://doi.org/10.1115/imece2015-53265},
+  year    = 2015,
+  month   = nov,
```

Comment:
The conference happened in Nov 15, but the article was first published online Mar 2016.

Reply:
As a general rule, please do one thing at a time. It's impossible to look through thousands of lines of changes that are automatically generated if you're also making semantic changes. If you need to make a change like this, do it in a separate patch.

Reply:
It's in the second commit. I can easily turn it into a separate PR if you like.